src/main/java/soot/Soot.java

```diff
@@ -0,0 +1,42 @@
+package soot;
+
+import soot.manager.CommandManager;
+
+import java.util.Scanner;
+
+public class Soot {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+        CommandManager commandManager = new CommandManager();
+
+        boolean isBye = false;
+
+        drawLine(); //initial greeting
+        greetUser();
+
+        while (!isBye) {
+            line = in.nextLine(); //user input
+            drawLine();
+            isBye = commandManager.isInputBye(line);
+        }
+        greetGoodbye();
+    }
+
+    public static void drawLine() {
+        int LINE_LENGTH = 60;
+        for (int i = 0; i < LINE_LENGTH; i++) {
+            System.out.print("_");
+        }
+        System.out.println("");
+    }
+    public static void greetUser() {
+        System.out.println("Hello! I'm Soot, your personal chatbot companion :)");
```

Comment:
Try not to use magic literals e.g. magic string, magic numbers. Consider goving it a meaning by creating a constant and use the constant in your code.